Source: Gentoo 
Upstream: No
Reason: Set widevine as available on linux

description: indicate that widevine continues to be supported on linux
author: Gregory M. Turner
origin: https://gitweb.gentoo.org/repo/gentoo.git/commit/www-client/chromium/files?id=09b804516320eee06930303870cd68008aac8a8a

--- third_party/widevine/cdm/BUILD.gn.old
+++ third_party/widevine/cdm/BUILD.gn
@@ -11,7 +11,7 @@ import("//third_party/widevine/cdm/widevine.gni")
 # Internal Cast builds set enable_widevine=true to bring in Widevine support.
 # TODO(xhwang): Support component updated CDM on other platforms and remove this
 # assert.
-assert(!enable_widevine || is_win || is_mac || is_chromecast,
+assert(!enable_widevine || is_win || is_mac || is_chromecast || is_linux,
        "Component updated CDM only supported on Windows and Mac for now.")
 
 widevine_arch = current_cpu
